lib/install/phases/bootstrap.go

```diff
@@ -203,6 +204,11 @@ func (p *bootstrapExecutor) configureSystemDirectories() error {
 		filepath.Join(stateDir, "secrets"),
 		filepath.Join(stateDir, "backup"),
 	}
+	p.Info(spew.Sdump(p.Application))
```

Comment:
Debugging aid or necessary?

Reply:
PR was an in draft stage :).
Now it is open for review, can you reviet it again? @a-palchikov